mirdata/datasets/haydn_op20.py

```diff
@@ -0,0 +1,383 @@
+"""haydn op20 Dataset Loader
+
+This dataset has as dependency the Music21 library.
+
+.. admonition:: Dataset Info
+    :class: dropdown
+
+    This dataset accompanies the Master Thesis from Nestor Napoles. It is a manually-annotated corpus of harmonic
+    analysis in harm syntax.
+
+    The dataset contains the following scores:
+    Haydn, Joseph
+    1. E-flat major, op. 20 no. 1, Hob. III-31
+    I. Allegro moderato
+    II. Menuetto. Allegretto
+    III. Affettuoso e sostenuto
+    IV. Finale. Presto
+    2. C major, op. 20 no. 2, Hob. III-32
+    I. Moderato
+    II. Capriccio. Adagio
+    III. Menuetto. Allegretto
+    IV. Fuga a 4 soggetti
+    3. G minor, op. 20 no. 3, Hob. III-33
+    I. Allegro con spirito
+    II. Menuetto. Allegretto
+    III. Poco adagio
+    IV. Finale. Allegro molto
+    4. D major, op. 20 no. 4, Hob. III-34
+    I. Allegro di molto
+    II. Un poco adagio e affettuoso
+    III. Menuet alla Zingarese & Trio
+    IV. Presto e scherzando
+    5. F minor, op. 20 no. 5, Hob. III-35
+    I. Allegro moderato
+    II. Menuetto
+    III. Adagio
+    IV. Finale. Fuga a due soggetti
+    6. A major, op. 20 no. 6, Hob. III-36
+    I. Allegro di molto e scherzando
+    II. Adagio. Cantabile
+    III. Menuetto. Allegretto
+    IV. Fuga a 3 soggetti. Allegro
+"""
+
+import os
+from typing import Any, BinaryIO, Dict, Optional, TextIO, Tuple, List
+
+import numpy as np
+
+from mirdata import core, io, jams_utils, download_utils
+import music21
+
+from mirdata.annotations import KeyData, ChordData
+
+BIBTEX = """
+@dataset{nestor_napoles_lopez_2017_1095630,
+  author={N\'apoles L\'opez, N\'estor},
+  title={{Joseph Haydn - String Quartets Op.20 - Harmonic Analysis Annotations Dataset}},
+  month=dec,
+  year=2017,
+  publisher={Zenodo},
+  version={v1.1-alpha},
+  doi={10.5281/zenodo.1095630},
+  url={https://doi.org/10.5281/zenodo.1095630}
+}"""
+
+REMOTES = {
+    "all": download_utils.RemoteFileMetadata(
+        filename="haydnop20v1.3_annotated.zip",
+        url="https://github.com/napulen/haydn_op20_harm/releases/download/v1.3/haydnop20v1.3_annotated.zip",
+        checksum="1c65c8da312e1c9dda681d0496bf527f",
+        destination_dir=".",
+    )
+}
+DOWNLOAD_INFO = ""
```

Comment:
Since there's no text, you can delete this line!

Reply:
done!